embedchain/vectordb/weaviate.py

```diff
@@ -206,14 +208,15 @@ def query(self, input_query: List[str], n_results: int, where: Dict[str, any], s
         :param skip_embedding: A boolean flag indicating if the embedding for the documents to be added is to be
         generated or not
         :type skip_embedding: bool
-        :return: Database contents that are the result of the query
-        :rtype: List[str]
+        :return: The content of the document that matched your query, url of the source, doc_id
+        :rtype: List[Tuple[str,str,str]]
         """
         if not skip_embedding:
             query_vector = self.embedder.embedding_fn([input_query])[0]
         else:
             query_vector = input_query
         keys = set(where.keys() if where is not None else set())
+        data_fields = ["text"]  # , "url", "doc_id"
```

Comment:
The comment here seems weird. Can you please fix it?

Reply:
Okay, I will need the changes from @rupeshbansal added for weaviateDB to support query function to return source url. Reverting it to work as before.